game/src/info/trip.rs

```diff
@@ -70,7 +70,7 @@ pub fn ongoing(
     {
         col.push(Widget::custom_row(vec![
             Widget::custom_row(vec![Line("Trip time").secondary().into_widget(ctx)])
-                .force_width_pct(ctx, col_width),
+                .force_width_abs_pct(ctx, col_width),
```

Comment:
nomenclature: "abs" as in an **abs**olute percent of the window size vs "rel" as in a percent **rel**ative to it's parent.

Open to suggestion on naming alternatives...

Reply:
`abs` / `rel` is standard enough. Window size vs parent isn't clear from the name, so if it's different between the two, how about `force_width_abs_pct_{parent, window}`?

Reply:
Ah, I misunderstood your comment. It's always a percentage of **something**, so relative vs absolute indicate what. How do you feel about `force_width_pct_window` and `force_width_pct_parent`?

Reply:
I've adopted your naming suggestion.